### ISSUE TITLE
symbolic-ref not recognized

### ISSUE THREAD

**Issue by bigbtechnology** [SickRage/SickRage#2389]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: 
OS: Windows 10
What you did: disabled SickRage service, so I could use unc paths and network drive letters. Ran "c:\SickRage\Python\python.exe c:\SickRage\SickRage\SickBeard.py" as a batch file.
What happened: unc and network drives are recognized now. git was not found, so I specified it's path. And a constant message is in the top center of the sickrage webui, stating "You're using the 'symbolic-ref' is not recognized as an internal or external command, operable program or batch file. C:\SickRage\SickRage> branch. Please use 'master' unless specifically asked"
What you expected: Everything to work.
Logs:
```
2016-09-27 21:52:23 INFO     CHECKVERSION :: Version checking is disabled, not checking for the newest version
2016-09-27 21:52:23 INFO     TORNADO :: Starting SickRage on http://0.0.0.0:8081/
AAC:\SickRage\SickRage>] using 'C:\SickRage\SickRage\config.ini'
AA
AAoperable program or batch file.
2016-09-27 21:52:23 INFO     MAIN :: Starting SickRage ['symbolic-ref' is not recognized as an internal or external command,
2016-09-27 21:52:23 INFO     MAIN :: 74598: Unable to find IMDb show info in the database
2016-09-27 21:52:23 INFO     MAIN :: 81852: Unable to find IMDb show info in the database
2016-09-27 21:52:23 INFO     MAIN :: 311939: Unable to find IMDb show info in the database
2016-09-27 21:48:47 INFO     POSTPROCESSER :: Successfully processed
PASTE LOGS HERE
```

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [x] Post debug logs, either inline (for smaller logs) or using gist
- [x] Please wrap inline logs inside \`\`\` and \`\`\` for readability

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by neoatomic** [comment]
If you are not using the service/installer, You might be better off installing sickrage manually on Windows. 
That will probbebly solve the `symbolic-ref` and `Please use 'master' unless specifically asked` errors also. : 

https://github.com/SickRage/SickRage/wiki/SickRage-Windows-Installer#manual-installation-guides-for-windows

**Comment by bigbtechnology** [comment]
Thank you for the reply neoatomic.
The windows installer is how I installed SickRage. I disabled the service due to unc paths and network drives not being showing in the list of available root drives. unc paths also clear when i type them in manually, they won't stick. When i run python manually and load sickbeard.py from there, the unc's and the network paths work. but I get the issue listed above.

Symbolic links are also not effective for me, as the error logs shows that sickrage is unable to read or write to the symb linked path.

I may have to write it up as a issue with my MyBookLive nas drive. and look into other ways of scheduling transfers from a local sickrage post-process path to my nas.

I appreciate your time.
